### PR TITLE
Split qam-minimal-full* for SLE12

### PR DESCRIPTION
**Pull request by dzedro**
-without SYSTEM_ROLE=textmode & salt, nginx test

- Related ticket: https://progress.opensuse.org/issues/125423
- Verification run:
https://openqa.suse.de/tests/10869563 12-SP2
https://openqa.suse.de/tests/10867592 12-SP5 ppc64le
https://openqa.suse.de/tests/10869523 12-SP5 x86_64

### CODE REVIEW

**Review by grisu48 — APPROVED** [review]

File: schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+- installation/handle_reboot
+- installation/first_boot
+- console/system_prepare
+- console/hostname
+- console/prepare_test_data
+- console/consoletest_setup
+- qam-minimal/install_update
+- qam-minimal/update_minimal
+- qam-minimal/check_logs
+- '{{s390tools}}'
```
Comment (rfan1) [inline schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml:40]: Can we remove this ?

File: schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    test is first textmode after system update and RR update is done
+     "gnome" pattern is installed then console and x11 tests run
+    workaround:
+    schedule system_prepare also after install_patterns on 15-SP1 & SP2 ppc64le
+vars:
+  DESKTOP: 'textmode'
+schedule:
+- installation/bootloader_start
+- installation/welcome
+- '{{disk_activation}}'
```
Comment (rfan1) [inline schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml:14]: Can we remove this？

File: schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml (hunk 1 above)
Comment (dzedro) [reply schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml:40]: no, it's used for all architectures

File: schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml (hunk 2 above)
Comment (dzedro) [reply schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml:14]: no, it's used for all architectures

File: schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml (hunk 1 above)
Comment (dzedro) [reply schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml:40]: Ah it's ppc64le, sorry will remove that

Comment (dzedro) [reply schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml]: done

File: schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml (hunk 2 above)
Comment (dzedro) [reply schedule/qam/common/qam-minimal-full-ppc64le-sle12.yaml]: done

**Review by rfan1 — APPROVED** [review]
LGTM

**Comment by dzedro** [comment]
Thanks, just for the record this is the price when you improve/simplify something and don't test everything. Then it can/will hit hard like updates on all affected SLE12 versions with many failures. :scream_cat:

**Pull request merged**